src/DIRAC/FrameworkSystem/private/authorization/utils/Tokens.py

```diff
@@ -159,18 +159,21 @@ def get_expires_in(self):
         """
         return int(self.get("expires_in"))
 
-    def is_expired(self):
+    def is_expired(self, timeLeft: int = 0):
         """A method to define if this token is expired.
 
+        :param timeLeft: extra time
+
         :return: bool
         """
+        now = time.time() + timeLeft
```

Comment:
This `now` is not exactly "now" (just use another identifier).

Reply:
fixed